### PR TITLE
Development

### PR DESCRIPTION
**Pull request by ucanfil**
Tested in ie10, firefox latest, chrome latest.
Tested for various screen sizes like: Galaxy S5, iPhone 6/7/8/x, iPad/pro etc.
Added one major breakpoint at screen-width: 900px,
            3 small breakpoints at screen-widths: 650px, 460px, 380px respectively.
Added bit of javascript for IE compatibility issue, and one for smooth scrolling behaviour.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
-# Portfolio-Site-V2
```
Comment (felixgate13) [inline README.md]: double title unless this was intended.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en" data-useragent="Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.2; Trident/6.0)">
```
Comment (mporam) [inline index.html]: the useragent is added by js, it shouldnt be here by default

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,787 @@
+/*! normalize.css v8.0.0 | MIT License | github.com/necolas/normalize.css */
```
[the remark this answers is not in the document]
Comment (ucanfil) [reply css/style.css]: Splitted out.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,787 @@
...
+  min-width: 300px;
+}
+
+.project a {
+  width: 100%;
+  height: 100%;
+}
+
+.project1 {
+  background-image: url("/img/todo-medium_600.png");
```
[the remark this answers is not in the document]
Comment (ucanfil) [reply css/style.css]: 👍

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,787 @@
...
+nav {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+}
+
+[alt="business-logo"] {
```
[the remark this answers is not in the document]
Comment (ucanfil) [reply css/style.css]: Wanted to use attribute selector, no reason at all. It's now under .logo class selector.

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,787 @@
...
+  align-items: flex-start;
+}
+
+.top-nav li {
+  height: 30px;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
```
[the remark this answers is not in the document]
Comment (ucanfil) [reply css/style.css]: It does change the look of the links on top-nav and padding effects the borders when you hover over them.

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,787 @@
...
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-pack: distribute;
+  justify-content: space-around;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  max-width: 500px;
+  width: 100%;
```
[the remark this answers is not in the document]
Comment (ucanfil) [reply css/style.css]: Deleted 1st one.

File: index.html (hunk 2 above)
Comment (ucanfil) [reply index.html]: Done.

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,443 @@
...
+.top-nav a:hover,
+.navbar-bottom a:hover {
+  color: #00c9d3;
+  -webkit-transition: color 0.5s;
+  -o-transition: color 0.5s;
+  transition: color 0.5s;
+}
+
+.top-nav a:hover {
+  border: 1px solid #00c9d3;
```
[the remark this answers is not in the document]
Comment (ucanfil) [reply css/style.css]: border: 1px solid transparent; property added to anchor tags 👍

**Pull request merged**